### PR TITLE
WIP: Handle new tokens format from website

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 3f39a31832b863d4d56681edcea7e87815611989

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 3f39a31832b863d4d56681edcea7e87815611989

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 53f1e7fc2d5bac2b1a8a2820d68cc965696cc8d8

File: mergify_engine/branch_updater.py [hunk 1]
```diff
@@ -99,15 +97,39 @@ def update(pull, installation_id, method="merge"):
...
 
         return git("log", "-1", "--format=%H").decode().strip()
     except subprocess.CalledProcessError as e:  # pragma: no cover
-        LOG.error("update branch fail: %s", e.output, pull_request=pull,
-                  exc_info=True)
+        for message in AUTHENTICATION_FAILURE_MESSAGES:
+            if message in e.output:
+                raise AuthentificationFailure(e.output)
+        else:
+            LOG.error("update branch fail: %s", e.output, pull_request=pull,
```
Comment (jd) [inline mergify_engine/branch_updater.py]: `fail` -> `failed`

File: mergify_engine/branch_updater.py [hunk 2]
```diff
@@ -99,15 +97,39 @@ def update(pull, installation_id, method="merge"):
...
         git.cleanup()
+
+
+def update(pull, installation_id, method="merge"):
+    redis = utils.get_redis_for_cache()
+
+    subscription = sub_utils.get_subscription(redis, installation_id)
+
+    if not subscription:
+        LOG.error("subscription to update branch is missing")
```
Comment (jd) [inline mergify_engine/branch_updater.py]: `unable to update branch: subscription is missing`

**Comment by sileht** [comment]
Moved here: https://github.com/Mergifyio/mergify-engine/pull/385

**Pull request closed without merging**